### ISSUE TITLE
Apikana Maven plugin fails with Apikana > 0.7.4

### ISSUE THREAD

**Issue by thwint** [swisspush/apikana#164]
When using the Apikana maven plugin the build fails, when using an Apikana 0.7.5 or greater.

It seems that generating of json-schema-3/event.json is broken. This only happens when an API uses one or more other dependent API.

Version 0.7.5 generates the following lines in event.ts:
```
    "driverSignon": {
      "$ref": "../../../api-dependencies/ts//vehicleoperation-recording-api/target/api-dependencies/json-schema-v3/driver-signon.json",
      "description": "the current signed in Driver on the vehicle"
    },
```
The part `ts//vehicleoperation-recording-api/target/api-dependencies/` seems to be wrong as it does not point to the correct relative location on the filesystem.

Apikana 0.7.4 generates these lines correctly
```
    "driverSignon": {
      "$ref": "../../../api-dependencies/json-schema-v3/user-registration-api/driver-signon.json",
      "description": "the current signed in Driver on the vehicle"
    },
```
The following line in Apikana is responsible for the change:
https://github.com/swisspush/apikana/compare/v0.7.4...v0.7.5%23diff-8793a8289addf2b230a55060bd229d406b829d97cae2c8593e32a4b9fa6070a9R61

**Comment by lbovet** [comment]
The dependency management system changed during the transformation to full NPM project that started in 0.7.5 and lead to 0.8.0.
From this point, apikana-java is no more used and the compatibility with apikana-java is not guaranteed.
Restoring the compatibility with apikana-java is probably doable. You can submit a PR for that.

**Comment by Kusig** [comment]
Aehhhhh, we don't have Maven pom's anymore for Apikana. This would be a major killer problem.
Our SW depends heavily on Maven dependencies (around 100 API's)  inhouse but as well with other companies using the same. Such a change in Apikana is not acceptable at all and wasn't announced as well. Therefore Maven MUST be supported consistently in the future.

**Comment by lbovet** [comment]
Fixed by #166
